test/browser-only/__snapshots__/constructLayoutEngine-browser.test.js.snap

```diff
@@ -0,0 +1,91 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`constructLayoutEngine browser can successfully construct a layout engine and respond to routing events 1`] = `
+<body>
+  <div
+    id="single-spa-application:@org-name/header"
+  />
+  <div
+    id="single-spa-application:@org-name/footer"
+  />
+</body>
+`;
+
+exports[`constructLayoutEngine browser can successfully construct a layout engine and respond to routing events 2`] = `
+<body>
+  <div
+    id="single-spa-application:@org-name/header"
+  />
+  <div
+    id="single-spa-application:@org-name/app1"
+  />
+  <div
+    id="single-spa-application:@org-name/footer"
+  />
+</body>
+`;
+
+exports[`constructLayoutEngine browser can successfully construct a layout engine and respond to routing events 3`] = `
+<body>
+  <div
+    id="single-spa-application:@org-name/header"
+  />
+  <div
+    id="single-spa-application:@org-name/footer"
+  />
+  <div
+    id="single-spa-application:@org-name/app1"
```

Comment:
in a future PR I'll hopefully add logic to remove dom elements that are no longer active